sklearn/linear_model/_ridge.py

```diff
@@ -2523,14 +2572,24 @@ class RidgeCV(MultiOutputMixin, RegressorMixin, _BaseRidgeCV):
 
     Attributes
     ----------
-    cv_values_ : ndarray of shape (n_samples, n_alphas) or \
+    cv_results_ : ndarray of shape (n_samples, n_alphas) or \
             shape (n_samples, n_targets, n_alphas), optional
         Cross-validation values for each alpha (only available if
-        ``store_cv_values=True`` and ``cv=None``). After ``fit()`` has been
+        ``store_cv_results=True`` and ``cv=None``). After ``fit()`` has been
         called, this attribute will contain the mean squared errors if
         `scoring is None` otherwise it will contain standardized per point
         prediction values.
 
+        .. versionchanged:: 1.5
+            `cv_values_` changed to `cv_results_`.
+
+    cv_values_ : ndarray of shape (n_samples, n_alphas) or \
+            shape (n_samples, n_targets, n_alphas), optional
+
+        .. deprecated:: 1.5
+            `cv_values_` is deprecated in version 1.5 in favor of
+            `cv_results_` and will be removed in version 1.7.
+
```

Comment:
I don't think we need to document this, we could simply remove here.

Reply:
I think it makes sense to document deprecation of the constructor arguments since they're visible to users when they create the instance.